lib/api_guard/jwt_auth/refresh_jwt_token.rb

```diff
@@ -18,7 +23,7 @@ def refresh_tokens_for(resource)
       end
 
       def find_refresh_token_of(resource, refresh_token)
-        refresh_tokens_for(resource).find_by_token(refresh_token)
+        RefreshToken.where('expire_at > ?', Time.now.utc).where(token: refresh_token).first
```

Comment:
This query will make old refresh tokens invalid since this will skip records that doesn't have `expire_at` value. We need to update the query to handle `NULL` value in `expire_at` field.

Reply:
@Gokul595 I can't remember now (or figure out) how to run the tests in the dummy app before I commit my changes. Can you please let me know and I'll get the tests passing and update this. thanks!